### ISSUE TITLE
Feeds that don't work anymore

### ISSUE THREAD

**Issue by saamerm** [planetxamarin/planetxamarin#719]
**Describe the bug**
While debugging through the preview page of the solution, I noticed that several feeds return errors of some sort. So I decided to list them here and who they belong to:
1. https://www.basdecort.com/rss : @basdecort 
2. https://mallibone.com/feed/ : @mallibone 
3. https://blog.mzikmund.com/feed/?lang=en_us : @MartinZikmund 
4. https://xamarinhowto.com/feed/ : @xamarinhowto

There are a few others but the reason for failure might be related to libraries we are using 

**To Reproduce**
1. Clone the repo
2. Build the solution with debugging, and have it launch in a browser
3. Place an exception catch point in the breakpoints tab
4. Go to the Preview page of the website, and check the feedUri of the exceptions that are raised

**Expected behavior**
There should be no error raised

**Screenshots**
When you take the URI causing the exception and put it in the browser, you can verify the issue is happening. Here's an example of what displays instead of the RSS feed:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/8262287/111920099-9ae49880-8a63-11eb-9778-4927603bbdd1.png">

**Comment by MartinZikmund** [comment]
Yes, sorry, the feed does work, but loads terribly long. It is some weird Wordpress issue, and I am working on rewriting the website to custom ASP.NET Core implementation instead. Hopefully soon to be finished.

**Comment by mallibone** [comment]
Thanks for pointing this out @saamerm - migrated my blog lately and must have missed this due to caching holding on to the old blog 🙈
